Add instructions for running the tutes (#120)
tskit-dev/tutorials: intro.md

such as for [msprime](https://github.com/tskit-dev/msprime/discussions) and
[tsinfer](https://github.com/tskit-dev/tsinfer/discussions).

(sec_intro_running)=

## Running tutorial code

> It is not necessary to run code in the tutorials yourself: tutorials can be followed by
> simply reading each page. However, it is also possible to run the tutorial code
> on your own computer, which will allow you to experiment with the examples provided.

Reviewer: Bit wordy for me, how about

```suggestion
It is possible to run the tutorial code
on your own computer, if you wish. This will allow you to experiment with the examples provided.
```

Author: Sure, sounds good to be. Committed.
